core/src/main/java/org/moditect/jfrunit/JfrPredicate.java

```diff
@@ -0,0 +1,41 @@
+package org.moditect.jfrunit;
+
+import java.util.function.Predicate;
+
+import jdk.jfr.consumer.RecordedEvent;
+
+public class JfrPredicate {
+    private Predicate<RecordedEvent> predicate;
+    private String attributeName;
+    private String value;
```

Comment:
Can they be final?

Reply:
Sure, actually idea suggests to convert it into a record